### DISCUSSION TITLE
Overview of planned time

### DISCUSSION THREAD

**Discussion by niels-przybilla** [kimai/Paid-plugins#150] [Questions & Answers] [answered]
Hello,

is there any overview in Kimai how much worktime is already planned for a user with task estimations?

THX and best wishes

Niels

**Comment by kevinpapst** [comment, marked as answer]
The task management plugin ships two extra reports. 

![image](https://github.com/kevinpapst/kimai-plugins/assets/533162/5eb46629-b0b5-4775-a8d8-b49db6db73c1)
